client/components/room/Room.jsx

```diff
@@ -1,41 +1,50 @@
-import io from 'socket.io-client';
 import { connect } from 'react-redux';
+import { Redirect } from 'react-router-dom';
 import React from 'react';
 import Video from './Video';
 import Workspace from './Workspace';
 import Chat from './Chat';
 
+import io from 'socket.io-client';
 const server = location.origin;
 const socket = io(server);
 
 class Room extends React.Component {
   constructor(props) {
     super(props);
+    this.state = {
+      redirect: false
+    };
   }
 
   componentDidMount() {
     socket.emit('join room', this.props.roomName);
+    socket.on('full', (room) => {
```

Comment:
I'm a little confused how this pull request is different from the one you requested under `Implement/add a test to chat and workspace`. This piece of code handling redirects was included in the other request and It seems like we've all looked at the other one...do we need to look at this one too or shall we just merge the first one @dshelters?